### ISSUE TITLE
where do I place it for windows?

### ISSUE THREAD

**Issue by sublines** [edwardloveall/atom-replacement-icon#4]
not really an issue, just an information request :)
thanks

**Comment by edwardloveall** [comment]
I'll be honest, I'm not sure. Can you try these and see if any of them work? http://www.wikihow.com/Change-the-Icon-for-an-Exe-File

**Comment by sublines** [comment]
Thanks, i looked at it but found a much easier way. 
For anyone interested: 
1. Right click the shortcut and select Properties
2. Select "Change Icon..."
This only works with shortcuts, not with the original exe file.

**Comment by edwardloveall** [comment]
Yeah I found that too but it shows up the with the old icon in the taskbar. I'll leave this open. If I or someone else knows a better way, feel free to post it and I'll add it to the readme.
